app/controllers/imports_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 class ImportsController < ApplicationController
   before_action :set_import, except: %i[ index new create ]
+  protect_from_forgery with: :null_session, include: %i[ upload_csv ]
```

Comment:
I don't think this is the best practice, however was unsure how to make it work from a JS http request.